bridgenode/initrestore.go

```diff
@@ -28,43 +23,39 @@ func initBridgeNodeState(
 	// If either is incomplete or not complete, they're both removed and made
 	// anew
 	// Check if the offsetfiles for both rev*.dat and blk*.dat are present
-	if util.HasAccess(util.OffsetFilePath) {
-		knownTipHeight, err = restoreLastIndexOffsetHeight(
-			offsetFinished)
+	if util.HasAccess(cfg.utreeDir.offsetDir.offsetFile) {
+		knownTipHeight, err = restoreLastIndexOffsetHeight(cfg.utreeDir.offsetDir, offsetFinished)
 		if err != nil {
 			err = fmt.Errorf("restoreLastIndexOffsetHeight error: %s", err.Error())
 			return
 		}
 	} else {
 		fmt.Println("Offsetfile not present or half present. " +
 			"Indexing offset for blocks blk*.dat files...")
-		knownTipHeight, err = createOffsetData(p, dataDir, offsetFinished)
+		knownTipHeight, err = createOffsetData(cfg, offsetFinished)
 		if err != nil {
 			err = fmt.Errorf("createOffsetData error: %s", err.Error())
 			return
 		}
 		fmt.Printf("tip height %d\n", knownTipHeight)
 	}
 
-	if cowForest {
-		if util.HasAccess(util.CowForestCurFilePath) {
+	if cfg.cowForest {
```

Comment:
The if/else blocks look very similar and could be cleaned up. They only differ in the file that gets passed to `util.HasAccess`. Maybe something for another PR, since this PR did not introduce this.

Reply:
Ah yeah not the best. I'll clean this up with this PR since it is sorta a cleanup PR anyways

Reply:
Fixed with a `forestType` enum